.github/workflows/build.yaml

```
@@ -25,6 +25,10 @@ on:
         description: 'Comma separated list of images to skip building, example with all possible images: mlrun,api,base,models,models-gpu,jupyter,test'
         required: false
         default: ''
+      disable_cache:
+        description: 'Whether to force cache miss, the default is no because doing so significantly increases build time (set any character to force this action)'
+        required: false
+        default: ''
```

The wording of the help message is a bit clunky, but I believe it's mainly because the input is negated. How about:?
```suggestion
      build_from_cache:
        description: 'Whether to build images from cache or not. Defaults to 'true' because not doing so significantly increases build time'
        required: false
        default: 'true'
```